src/source/location.d

```diff
@@ -45,19 +46,15 @@ public:
 		return start.isMixin();
 	}
 
-	auto spanTo(Location end) in {
-		import std.conv;
-		assert(stop.offset <= end.stop.offset,
-		       to!string(stop.offset) ~ " > " ~ to!string(end.stop.offset));
-	} do {
+	auto spanTo(Location end)
+			in(stop.offset <= end.stop.offset,
+			   to!string(stop.offset) ~ " > " ~ to!string(end.stop.offset)) {
```

Comment:
Maybe it is out of scope, but it seems that format would be called for here.

Reply:
#347 implements this as a separate PR

Reply:
Thanks!